settings.json.sample

```diff
@@ -0,0 +1,67 @@
+{
+    "server":{
+        "hostname":"test",
+        "scheme":"https",
+        "port":"",
+        "ssh.key_private":"/home/whoami/.ssh/id_hudson_dsa",
+        "ssh.username":"root"
+    },
+    "ldap":{
+        "ldaphost":"test",
+        "ldapuser":"test",
+        "ldappwd":"test",
+        "basedn":"test",
+        "grpbasedn":"test"
+    },
+    "robottelo":{
+        "project":"sat",
+        "locale":"en_US.UTF-8",
+        "remote":"0",
+        "smoke":"0",
+        "upstream":"1",
+        "verbosity":"2",
+        "screenshots.base_path":"/tmp/robottelo/screenshots/",
+        "virtual_display":"0",
+        "window_manager_command":"test"
+    },
+    "tests":{
```

Comment:
Let's suppose:

```json
{
  ...
  "clients": {...},
}
```

and

```json
{
  ...
}
```

In the first robottelo should assume that the clients feature is enabled and the second it is not available. With that said there is no need to define a boolean telling to run or skip some tests based on the features.